scripts/add_existing_baseyear.py

```diff
@@ -104,7 +104,7 @@ def add_existing_renewables(df_agg, costs):
                 n.generators.loc[gens, "bus"]
             ).sum()
 
-        nodal_df = df.loc[n.buses.loc[elec_buses, "country"]]
```

Comment:
How are the NaN values handled here? Shouldn't they be set to zero?

Reply:
I agree that this is only a quick fix. So this is not meant to pull into the master. I do not get the error anymore, so I cannot investigate the error.

Reply:
> How are the NaN values handled here? Shouldn't they be set to zero?

True. I added fill_value=0 to reindex()